src/pages/ResultsListPage.js

```diff
@@ -101,8 +101,11 @@ class ResultsListPage extends BaseResultsPage {
     }
 
     renderPersonalisationLink(): ReactElement {
+        const current = this.props.location.pathname;
+        const separator = current.endsWith("/") ? "" : "/";
```

Comment:
Yes - we need eg `/category/housing/personalise/summary`, not `/category/personalise/summary`